pkg/errors/stacktrace.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package errors
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+)
+
+// stack holds the address of
+type stacktrace []uintptr
+
+// New captures a stack trace. skip specifies the number of frames to skip from
+// a stack trace. skip=0 records stack.New call as the innermost frame.
```

Comment:
`skip` isn't passed as an argument here.

Reply:
We won't need it. Removing the comments. The `pkg/errors` uses hardcoded skip=3. I think that's sufficient here too.